File: python/8 kyu/259.py
# https://www.codewars.com/kata/57eadb7ecd143f4c9c0000a3/train/python

# Passed

def abbrev_name(name):
    first, last = name.split(' ')
    initials = first[0].upper() + "." + last[0].upper()
    return initials

output = abbrev_name("Sam Harris")
print(output)

File: nodejs/7 kyu/259.js
// https://www.codewars.com/kata/5a05fe8a06d5b6208e00010b/train/javascript

// Passed

function seqToOne(n) {
  let sequenceArr = [];
  let currentNum = n;
  while (currentNum !== 1) {
    sequenceArr.push(currentNum);
    currentNum = currentNum < 1 ? currentNum + 1 : currentNum - 1;
  }

  sequenceArr.push(currentNum);

  return sequenceArr;
}

const output = seqToOne(100);
console.log(output);